values.yaml

```diff
@@ -184,3 +184,8 @@ config:
     enabled: false
     token: ""
 
+## Here you can set extra settings from https://docs.gitea.io/en-us/config-cheat-sheet/
```

Comment:
since this only works with an RC version of gitea can you add a comment aboiut version compatibility. Otherwise LGTM

Reply:
the `extra_config` settings works for all versions, just the two example config keys that were in the `values.yaml` were from v1.11.0-rc1. I changed that now.